src/instructlab/model/evaluate.py

```diff
@@ -334,6 +344,20 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--backend",
+    type=str,
+    default=backends.VLLM,
+    show_default=True,
+    help="serving backend to use for evaluation. Options are vllm and llama-cpp.",
+)
+@click.option(
+    "--max-ctx-size",
```

Comment:
Since this only applies to llama-cpp, can we remove this flag and just read from the serve config?

Reply:
💯

Reply:
The flow we are doing for each new command is the following:

config to...
flag overrides to...
libraries

I really am trying to drive this home from a design perspective. If we allow cases like this to slip because of behind these scenese stuff, it kind of hurts the overall new design of the CLI.

Reply:
I still have the concern that we don't want a user to set this for eval ever and so I can't see the use case for having an option.  And if it's set in the config we need to override it.

Reply:
but I really think we can't manually override a users config, it needs to be the source of truth. So what I am trying to do here is say "hey, set the right ctx size in your config or else pass us an override.".

Starting to do config editing behind the scenes is a slippery slope IMO.

Thats why I had that warning log originally, which I think if we add back that would make sense

Reply:
and if cfg is a source of truth as @alimaredia suggested above, then having an override fits the scheme the rest of the CLI is following